### DISCUSSION TITLE
Repeater Screen1 to Screen 2

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#11902] [Help] [answered]
Hello,

I want to give When clicking on the table on Screen1 page Then the information will be displayed on another screen.

I can make it happen when I click on the Table row on Screen1 page. Then go to another screen. But the information on the page Screen 2 doesn't change depending on the row I choose.
![image](https://github.com/Budibase/budibase/assets/93240913/0eecbd82-60fc-420c-9f9a-5d740cf2a6c1)

![image](https://github.com/Budibase/budibase/assets/93240913/62e7b8ae-57dc-4394-9b49-5eb9b9cca73f)

Thank you,

**Comment by andz-bb** [comment]
Hey @LTCTHITA 

You should be able to [use URL variables here. ](https://docs.budibase.com/docs/url-parameters)

Update Screen 2's route to be something like `/show/:id`. Now in the _Navigate To_ action in Screen 1, you can add the relevant ID as a binding to the end of the route. It would be something like Navigate To `/show/{{ Clicked row.id }}` or whichever binding has the ID you need.

Now in Screen 2, you can filter any Data Provider using that URL variable. That variable will be available in the binding `{{ URL.id }}`

Hope this helps!

**Reply by LTCTHITA** [reply]
Hey @andz-bb 

I can't do it

I have done as you suggested. But it doesn't give me any information. I don't know if I made a mistake or skipped a step. Where are you going or not?

Screen 1
![1 1](https://github.com/Budibase/budibase/assets/93240913/09868812-5e32-4509-b9d8-8cfd7dec4a36)

Screen 2
![1 2](https://github.com/Budibase/budibase/assets/93240913/7a599af5-0867-4e78-9fa7-1ba83444071d)

preview
![1](https://github.com/Budibase/budibase/assets/93240913/8ed6cacc-04ce-4de3-b88c-10fee139d62f)

Thank you,

**Reply by melohagan** [reply]
Hey @LTCTHITA your binding in the first screenshot is incorrect. It should be: `{{ Clicked row._id }}`

**Comment by LTCTHITA** [comment]
Hey @melohagan 
Have done as you said. But it seems like they still can't find each other. 
Screen1 : 
![image](https://github.com/Budibase/budibase/assets/93240913/fee369f1-45d9-4799-8281-ed44a1514636)


Screen2 : 
![image](https://github.com/Budibase/budibase/assets/93240913/0baa4b51-113e-4553-8611-38fc7da35d0c)
 


Preview : 
![image](https://github.com/Budibase/budibase/assets/93240913/246e0af1-8d9f-4654-80c2-ccbce6ca5495)

Thank you,

**Reply by melohagan** [reply, marked as answer]
Ah sorry only noticing now as well, you need to replace the `:id`

So the URL would look like: `/add-calibration/discharge/{{ Clicked row._id }}`
